metricbeat/module/docker/container/data.go

```diff
@@ -71,7 +71,9 @@ func eventMapping(r mb.ReporterV2, cont *types.Container, dedot bool) {
 func extractIPAddresses(networks *types.SummaryNetworkSettings) []string {
 	ipAddresses := make([]string, 0, len(networks.Networks))
```

Comment:
With this change `ipAddresses` is now not anymore `len(networks.Networks)`. Is it mainly in testing that it might be empty or a common case? Thinking which use case we should optimise for.

Reply:
I though about this and I think that in most of the cases `ipAddresses` is still going to be `len(networks.Networks)`.
The most common case where this is not going to be true is when the container is using the host network, in that case we would be creating an array of capacity one that we are not going to use.